foreman_certs/config/foreman-certs-answers.yaml

```
@@ -0,0 +1,6 @@
+certs:
+  generate: true
+  regenerate: true
+  deploy: false
+  group: foreman
```

```suggestion
  group: root
```
Wouldn't that make more sense, making it work in environments where there is no `foreman` group?
(OTOH it probably doesn't matter much as long `deploy` is `false`?)